test/helpers/e2e.go

```diff
@@ -60,20 +67,26 @@ type L1Environment struct {
 // L2Environment contains setup results for L1 network.
 type L2Environment struct {
 	CommonEnvironment
-	GERContract      *globalexitrootmanagerl2sovereignchain.Globalexitrootmanagerl2sovereignchain
-	AggoracleSender  aggoracle.ChainSender
-	EthTxManagerMock *EthTxManager
+	GERContract                *globalexitrootmanagerl2sovereignchain.Globalexitrootmanagerl2sovereignchain
+	AggOracleCommitteeContract *aggoraclecommittee.Aggoraclecommittee
+	AggoracleSender            aggoracle.ChainSender
+	EthTxManagerMock           *EthTxManager
 }
 
 type EnvironmentConfig struct {
 	L1RPCClient aggkittypes.RPCClienter
 	L2RPCClient aggkittypes.RPCClienter
+	AggoraclecommitteeConfig
```

Comment:
Why we are embedding this struct? IMO it is cumbersome and would make sense to have a field associated. WDYT?


```suggestion
	aggoracleCommitteeCfg AggoraclecommitteeConfig
```

Reply:
done